timApp/defaultconfig.py

```diff
@@ -31,6 +31,10 @@
 # If None, link to the document is not shown
 ACCESSIBILITY_STATEMENT_DOC = "tim/saavutettavuusseloste"
 
+# Path to TIM document containing the Terms of service The link to the document is shown in page footer.
+# If None, link to the document is not shown
+TERMS_OF_SERVICE_DOC = "tim/kayttoehdot"
```

Comment:
Ehdotan ehkä jättämään sen alussa tyhjäksi, sillä oletuksena varmaan palvelulla ei ole käyttöehtoja (esim. meillä testi-TIMit, yms).
Myös tätä on helpompaa ottaa käyttöön nyt, koska linkki pitäisi näyttää vasta elokuusta alkaen :)

```suggestion
TERMS_OF_SERVICE_DOC = None
```